ethereum/statetransition/src/jmh/java/tech/pegasys/teku/statetransition.validation.signatures/AggregatingAttestationPoolBenchmark.java

```diff
@@ -55,25 +63,27 @@
 @Fork(1)
 @State(Scope.Thread)
 public class AggregatingAttestationPoolBenchmark {
-  private static final Spec SPEC = TestSpecFactory.createMainnetDeneb();
+  private static final Spec SPEC = TestSpecFactory.createMainnetElectra();
 
   // pool dump can be created via something similar to
-  // https://github.com/tbenr/teku/commit/bd37ec8f5c6ce02edb3e375a1561e1d934b7d191
+  // https://github.com/tbenr/teku/commit/08ab0e3ac6e71a9d6340ae30757c63f922f2c968
   // state and actual block can be obtained the usual ways
 
   // a reference file can be obtained here
   // https://drive.google.com/file/d/139bA7r88riFODZ7S0FpvtO7hmWmdC_XC/view?usp=drive_link
-  private static final String STATE_PATH =
-      "BeaconStateDeneb_3630479_03664f196162fb81a4406c508674dd1ede09b883d37d0f3d0f076897f68741d2.ssz";
+  private static final String STATE_PATH = "/Users/tbenr/Documents/parentBlockState_3816770.ssz";
 
   // a reference file can be obtained here
   // https://drive.google.com/file/d/1I5vXK-x8ZH9wh40wNf1oACXeF_U3to8J/view?usp=drive_link
-  private static final String POOL_DUMP_PATH = "attestations_3630479.multi_ssz";
+  private static final String POOL_DUMP_PATH =
+      "/Users/tbenr/Documents/attestations_3816773.multi_ssz";
+  private static final UInt64 SLOT = UInt64.valueOf(3816773);
 
   // a reference file can be obtained here
   // https://drive.google.com/file/d/1PN0OToyNOV0SyjeQaS7oF3J4cKbmy1nX/view?usp=drive_link
-  private static final String ACTUAL_BLOCK_PATH =
-      "block-3630480-e652bd51c7e4e528fea0728a3ad96f86ceb92e9daa227f315e96a9884ceb187b.ssz";
+  private static final Optional<String> ACTUAL_BLOCK_PATH =
+      Optional.of(
+          "/Users/tbenr/Documents/block-3816773-456c9819a4c1e792ba8b1f71119628aed2a4d1e0d2c66199fbc763832937421b.ssz");
```

Comment:
remove path

Reply:
I knew I would forget it

Reply:
just set the running directory to documents and you dont have to set the path here :D